src/pages/Home.vue

```diff
@@ -84,8 +86,9 @@ export default defineComponent({
       Cards: Cards,
       Calculators: Calculators,
       Credits: Credits,
+      Equipment: Equipment,
     };
-    const defaultTab = 'Characters';
+    const defaultTab = 'Equipment';
```

Comment:
Navigation is changing in redesign but I don't think this should be the default? This feature will probably get moved to the character page since it's related to each character rather than a standalone feature.